packages/nodes-base/nodes/MondayCom/MondayCom.node.ts

```diff
@@ -758,6 +803,144 @@ export class MondayCom implements INodeType {
 						responseData = responseData.data.move_item_to_group;
 					}
 				}
+				if (resource === 'boardSubItem') {
+					if (operation === 'create') {
+						const boardId = this.getNodeParameter('boardId', i);
+						const groupId = this.getNodeParameter('groupId', i) as string;
+						const itemId = this.getNodeParameter('itemId', i) as string;
+						const subItemName = this.getNodeParameter('subItemName', i) as string;
+						const additionalFields = this.getNodeParameter('additionalFields', i);
+
+						const body: IGraphqlBody = {
+							query: `
+									mutation ($itemId: ID!, $subItemName: String!) {
+										create_subitem (parent_item_id: $itemId, item_name: $subItemName) {
+											id
+										}
+									}
+							`,
+							variables: {
+								itemId,
+								subItemName,
+							},
+						};
+
+						responseData = await mondayComApiRequest.call(this, body);
+						responseData = responseData.data.create_subitem;
+					}
+					if (operation === 'get') {
+						await this.getNodeParameter('boardId', i);
```

Comment:
This is likely to cause an issue and doesn't appear to actually do anything.

Reply:
it actually filters the boards, so the user can choose from them (there is also one for the groups and the items)